alerting/src/test/kotlin/org/opensearch/alerting/MonitorRunnerServiceIT.kt

```diff
@@ -1771,4 +1792,12 @@ class MonitorRunnerServiceIT : AlertingRestTestCase() {
     private fun Map<String, Any>.objectMap(key: String): Map<String, Map<String, Any>> {
         return this[key] as Map<String, Map<String, Any>>
     }
+    private fun updateTriggersLimits() {
+        client().updateSettings(
+            AlertingSettings.TOTAL_MAX_ACTIONS_PER_TRIGGER.key, 20
+        )
+        client().updateSettings(
+            AlertingSettings.TOTAL_MAX_ACTIONS_PER_TRIGGER.key, 20
+        )
+    }
```

Comment:
Remove `updateTriggersLimits` as it is not needed. Also the same setting is being updated twice. As long as these settings get reset to default after the tests in `AlertingSettingsIT.kt`, there wont be any need to do this.